src/backend/executor/execUtils.c

```diff
@@ -1571,8 +1571,9 @@ void mppExecutorFinishup(QueryDesc *queryDesc)
 		pgstat_combine_from_qe(pr, primaryWriterSliceIndex);
 
 		/* get num of rows processed from writer QEs. */
```

Comment:
where is bi comment https://github.com/arenadata/gpdb/commit/08a4193d46b7fe8667feaf61b0b15871bcb97fc0#diff-b48b322902e8cb8cf66825110ae659fcb3f7a15ad84cef5d4b3c3349d4618eb4R2166-R2170
?